### PR TITLE
[UI] Setup storybook

### PR DESCRIPTION
**Pull request by machulan**
[Trello task](https://trello.com/c/IedeQUtn)
[Visual design](https://www.figma.com/file/0KvoxIcY6iFSdVqZhwpOEd/MathRandom_web?node-id=14%3A0)

Changes:
* setup storybook
* add button component
* add text input component

### CODE REVIEW

**Review by ShevtsovAnton — APPROVED** [review]

File: components/common/Button/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+StyledButton.defaultProps = {
+  isRounded: false,
+  type: 'button',
+};
+
+export type ButtonProps = StyledButtonProps & {
+  text: string;
+};
+
+export const Button: React.FC<ButtonProps> = ({ text, ...props }) => (
```
Comment (osymonchuk-clgx) [inline components/common/Button/Button.tsx]: ESLint: 'text' is missing in props validation(react/prop-types). Define args type as ButtonProp.

File: components/common/Button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    outline: 0;
+  }
+`;
+
+StyledButton.defaultProps = {
+  isRounded: false,
+  type: 'button',
+};
+
+export type ButtonProps = StyledButtonProps & {
```
Comment (osymonchuk-clgx) [inline components/common/Button/Button.tsx]: It's better to move any other not-React component exports to a separate file. https://nextjs.org/docs/basic-features/fast-refresh

File: components/common/Input/TextInput.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  name: string;
+  label?: string;
+  isRequired?: boolean;
+  bottomText?: string;
+  hasError?: boolean;
+  hasValidValue?: boolean;
+  requiredMessage?: string;
+};
+
+export const TextInput: FC<TextInputProps> = ({ label, isRequired, bottomText, hasError, hasValidValue, requiredMessage, ...props }) => (
```
Comment (osymonchuk-clgx) [inline components/common/Input/TextInput.tsx]: Define args type

File: components/common/Button/Button.tsx (hunk 1 above)
Comment (machulan) [reply components/common/Button/Button.tsx]: fixed

File: components/common/Input/TextInput.tsx (hunk 3 above)
Comment (machulan) [reply components/common/Input/TextInput.tsx]: fixed

File: components/common/Button/Button.tsx (hunk 2 above)
Comment (machulan) [reply components/common/Button/Button.tsx]: moved

**Review by OlegSymonchuk — APPROVED** [review]

**Pull request merged**